example/lib/views/app_text_field.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+import 'package:master_utility/master_utility.dart';
+
+class AppTextField extends StatelessWidget {
+  final FocusNode focusNode;
+  final String label;
+  final bool isNextButton;
+  final bool isPrevious;
+  final bool showDoneKeyboard;
+  const AppTextField({
+    super.key,
+    required this.focusNode,
+    required this.label,
+    this.isNextButton = false,
+    this.isPrevious = false,
+    this.showDoneKeyboard = false,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.all(20.0),
+      child: TextFormField(
+        focusNode: showDoneKeyboard
+            ? (focusNode
+              ..addListener(() {
+                final hasFocus = focusNode.hasFocus;
+                if (hasFocus) {
+                  KeyboardOverlay.showOverlay(
+                    context: context,
+                    isNextButton: isNextButton,
+                    isPrevious: isPrevious,
+                    isShowButton: (isNextButton || isPrevious),
+                  );
+                } else {
```

Comment:
else will be called anytime as we assigned value to hasFocus?

Reply:
If it is focused, then it will view the Done View keyboard, and otherwise it will not view